client/my-sites/plugins/plugins-browser/index.jsx

```diff
@@ -256,26 +251,55 @@ const PluginsBrowser = React.createClass( {
 			this.props.translate( 'Writing', { context: 'Plugins suggested search term' } ),
 		];
 		// TODO: The Search does not get the search text when a suggested term is clicked
-		// TODO: do we still need any of the CSS in plugins-browser__main-header? It breaks the layout when the search field is open.
 		return (
-			<SectionNav selectedText={ this.props.translate( 'Suggested Searches', { context: 'Suggested searches for plugins' } ) }>
+			<SectionNav
+				selectedText={ this.props.translate( 'Suggested Searches', {
+					context: 'Suggested searches for plugins',
+				} ) }
+			>
 				<NavTabs label="Suggested Searches">
-					<NavItem path={ '/plugins' + site } selected={ false } >
+					<NavItem path={ '/plugins' + site } selected={ false }>
 						{ this.props.translate( 'All', { context: 'Filter all plugins' } ) }
 					</NavItem>
-					{ suggestedSearches.map( term => <NavItem
-						key={ `suggested-search-${ term }` }
-						path={ '/plugins' + site + '?s=' + term }
-					>
-						{ term }
-					</NavItem>
+					{ suggestedSearches.map( term =>
+						<NavItem key={ `suggested-search-${ term }` } path={ '/plugins' + site + '?s=' + term }>
+							{ term }
+						</NavItem>
 					) }
 				</NavTabs>
 				{ this.getSearchBox() }
 			</SectionNav>
 		);
 	},
 
+	getManageButton() {
+		const site = this.props.site ? '/' + this.props.site : '';
+		return (
+			<div className="plugins-browser__manage-button">
+				<Button compact href={ '/plugins/manage' + site }>
+					<Gridicon icon="cog" />
+					{ this.props.translate( 'Manage' ) }
+				</Button>
+			</div>
+		);
+	},
+
+	getPageHeaderView() {
+		if ( this.props.hideSearchForm ) {
+			return null;
+		}
+
+		const navigation = this.props.category ? this.getNavigationBar() : this.getSearchBar();
+		const manageButton = this.props.isJetpackSite && this.getManageButton();
```

Comment:
Why do we show this button in the sidebar for every site, but only show it here for Jetpack sites? I think we should do one or the other.